sqlglot/generator.py

```diff
@@ -423,8 +423,11 @@ def join_sql(self, expression):
         return f"{expression_sql}{op_sql} {this_sql}{on_sql}"
 
     def lambda_sql(self, expression):
+        a, b = (
```

Comment:
can we clean this up a bit

args = self.expressions(expression, flat=True)
args = f"({args})" if len(args) > 1 else args
return self.no_identify(f"{args} -> ...")

Reply:
Much neater, thanks. I wasn't happy with my code but no alternative came to mind.